### PR TITLE
Azure Defender EASM Management Plane SDK Public Preview

### PR DESCRIPTION
**Pull request by shadahmed-microsoft**
# EASM Mgmt Plane SDK for .NET

This PR is for the check-in of the Defender EASM Mgmt Plane SDK. The SDK was autogenerated from Swagger.

arcboard: https://github.com/Azure/azure-sdk/issues/5062

### CODE REVIEW

File: sdk/defendereasm/Azure.ResourceManager.DefenderEasm/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure DefenderEasm management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/defendereasm/Azure.ResourceManager.DefenderEasm/README.md:3]: Please add the description for your service here.

File: sdk/defendereasm/Azure.ResourceManager.DefenderEasm/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/defendereasm/Azure.ResourceManager.DefenderEasm/CHANGELOG.md:3]: Please specify the release date by replacing `Unreleased` , format is `YYYY-MM-DD`

File: sdk/defendereasm/Azure.ResourceManager.DefenderEasm/src/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("DefenderEasm")]
```
Comment (ArthurMa1978) [inline sdk/defendereasm/Azure.ResourceManager.DefenderEasm/src/Properties/AssemblyInfo.cs:6]: The value here should be `Microsoft.Easm`

File: sdk/defendereasm/Azure.ResourceManager.DefenderEasm/src/autorest.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+```yaml
+azure-arm: true
+csharp: true
+library-name: DefenderEasm
+namespace: Azure.ResourceManager.DefenderEasm
+require: https://github.com/Azure/azure-rest-api-specs/blob/23d88533ddfde4d1565a897fe95d42fb0d9333e5/specification/riskiq/resource-manager/readme.md
+tag: package-preview-2023-04
```
Comment (ArthurMa1978) [inline sdk/defendereasm/Azure.ResourceManager.DefenderEasm/src/autorest.md:11]: Please comment out this line, the sdk should always generate from the default tag

File: sdk/defendereasm/Azure.ResourceManager.DefenderEasm/src/autorest.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
```
Comment (ArthurMa1978) [inline sdk/defendereasm/Azure.ResourceManager.DefenderEasm/src/autorest.md:3]: Please run `eng\scripts\Export-API.ps1 defendereasm` to generate api file.

**Comment by ArthurMa1978** [comment]
@shadahmed-microsoft please fix the spelling error, add the `easm` into the [.vscode\cspell.json file](https://github.com/Azure/azure-sdk-for-net/blob/main/.vscode/cspell.json).

**Review by ArthurMa1978 — APPROVED** [review]

**Comment by thang-bit** [comment]
@ArthurMa1978 I noticed that this PR was merged. From this doc: https://eng.ms/docs/products/azure-developer-experience/develop/sdk-release, the next step will be requesting a release for this dotnet sdk, right?

**Pull request merged**